Practice1/src/css/header.css

```diff
@@ -0,0 +1,174 @@
+.page-header {
+    position: relative;
+}
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 63px;
+    padding: 25px 0px 0px 51px;
+    width: calc(100% - 92px);
+}
+
+.nav-img {
+    position: absolute;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+}
+
+.nav .btn-tag {
+    display: flex;
+    color: var(--white);
+    align-items: center;
+}
+
+.nav .btn-tag .btn-login {
+    padding-right: 20px;
+}
+
+.nav .btn-tag .btn-lyp {
+    border: 1px solid var(--white);
+    padding: 10px 10px;
+    width: 120px;
+}
+
+.nav .logo {
+    color: var(--black);
+    display: flex;
+    font-size: 10px;
+    align-items: center;
+}
+
+.nav .nav-input {
+    font-size: 20px;
+    color: var(--white);
+    padding-left: 10px;
+}
+
+.search {
+    display: flex;
+    height: 100px;
+    width: calc(100% - 115px);
+    justify-content: center;
+    margin: 0px auto;
+    margin-top: 306px;
+    margin-bottom: 100px;
```

Comment:
Please double-check your margin here from lines 56 to 58.

Reply:
Because the margin has been set to '0px auto', other margins like top or bottom I have to set individually to run